src/matchbox/server/postgresql/utils/db.py

```diff
@@ -50,6 +57,44 @@ def get_model_subgraph(engine: Engine) -> rx.PyDiGraph:
     return G
 
 
+def get_data_subgraph(engine: Engine) -> rx.PyDiGraph:
```

Comment:
Is this worth testing? By setting up the DB at the index level, and then checking against a manually constructed expectation object. Though every one of these tests that tear down and set up the DB takes some time to run

Reply:
Untested code in the codebase in main is tech debt. I don't mind if it goes on a branch or in a ticket or on Confluence